City-of-Pori/pori-kada
# Initial domain configuration

Add domains for pori.fi, visitpori.fi and kalenteri.pori.fi

The domains are not on variable, they are on domain-table on db so we need this change to be present in the feature.

To test:
1. drush cc all
2. Check from domains module configuration that the correct domains are available
3. Check that on content add form you are able to post content to these different domains.

More information on this PR from @sihv
